app/models/user.rb

```diff
@@ -206,12 +206,12 @@ def ldap_group
   end
   alias_method :miq_group_description, :ldap_group
 
-  def role_allows?(options = {})
-    Rbac.role_allows?(options.merge(:user => self))
+  def role_allows?(**options)
+    Rbac.role_allows?(:user => self, **options)
   end
 
-  def role_allows_any?(options = {})
-    Rbac.role_allows?(options.merge(:user => self, :any => true))
+  def role_allows_any?(identifiers:)
+    Rbac.role_allows?(:user => self, :any => true, :identifiers => identifiers)
```

Comment:
Is it simpler to define it like this, so that the Rbac class owns the interface?

```suggestion
  def role_allows_any?(**options)
    Rbac.role_allows?(:user => self, :any => true, **options)
```

Reply:
I agree with the premise, there are quite a few callers and was trying to nail down the actual interface.

I will change